### PR TITLE
Add server option to unbound plugin

### PR DESCRIPTION
**Pull request by mcfedr**
This allows unbound plugin to connect to other servers

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/unbound/unbound.go [hunk 1]
```diff
@@ -54,9 +60,18 @@ func (s *Unbound) SampleConfig() string {
 }
 
 // Shell out to unbound_stat and return the output
-func unboundRunner(cmdName string, Timeout internal.Duration, UseSudo bool) (*bytes.Buffer, error) {
+func unboundRunner(cmdName string, Timeout internal.Duration, UseSudo bool, Server string) (*bytes.Buffer, error) {
 	cmdArgs := []string{"stats_noreset"}
 
+	if Server != "" {
+		// Unbound control requires an IP address, and we want to be nice to the user
+		serverIp, err := net.LookupIP(Server)
```
Comment (danielnelson) [inline plugins/inputs/unbound/unbound.go]: I think you will need to `net.SplitHostPort` and pass in only the host here.

File: plugins/inputs/unbound/unbound.go [hunk 2]
```diff
@@ -42,6 +44,10 @@ var sampleConfig = `
 
   ## Use the builtin fielddrop/fieldpass telegraf filters in order to keep/remove specific fields
   fieldpass = ["total_*", "num_*","time_up", "mem_*"]
+  
+  ## IP of server to connect to, read from unbound conf default, optionally '@port'
+  ## Will lookup IP if given a hostname
+  server = "127.0.0.1@8953"
```
Comment (danielnelson) [inline plugins/inputs/unbound/unbound.go]: Use `:` as separator

File: plugins/inputs/unbound/unbound.go (hunk 1 above)
Comment (danielnelson) [inline plugins/inputs/unbound/unbound.go]: Use a resolver that can accept a timeout.

**Comment by mcfedr** [comment]
@danielnelson How does it look now? I've been using it in production for last couple of weeks now

**Pull request merged**